### DISCUSSION TITLE
post/upload-azure-blob-storage-sas

### DISCUSSION THREAD

**Discussion by giscus** [markheath/soundcode-blog-comments#8] [Announcements]
# post/upload-azure-blob-storage-sas

Mark Heath's Development Blog

https://markheath.net/post/upload-azure-blob-storage-sas

**Comment by mbarmettler** [comment]
but this function "GetSharedAccessSignature" does not exist anymore on the Blob Object, since using Azure.Storage.Blobs Libraries (Sdk)

**Comment by markheath** [comment]
Sure, this blog post is five years old and Microsoft have brought out a newer (v12) SDK. I've written a bunch more posts on the newer blob storage SDKs (including covering some newer ways of generating SAS) that you can find [here](https://markheath.net/category/azure%20blob%20storage)

You can also follow [this tutorial on Microsoft learn](https://learn.microsoft.com/en-us/azure/storage/blobs/sas-service-create?tabs=dotnet)

**Comment by luisfleyva** [comment]
Awesome guide! 👏 Super clear walkthrough on generating upload-only SAS tokens and uploading via the Azure SDK or REST API. Love how you demonstrate both UploadFromFileAsync and raw HttpClient uploads—makes it easy to choose what fits best. Have you worked with SAS tokens and browser-based file uploads using JavaScript?
